core/shared/src/main/scala/org/typelevel/log4cats/LoggerFactory.scala

```diff
@@ -22,36 +22,10 @@ import scala.annotation.implicitNotFound
 Implicit not found for LoggerFactory[${F}].
 Information can be found here: https://log4cats.github.io/logging-capability.html
 """)
-trait LoggerFactory[F[_]] extends LoggerFactoryGen[F, SelfAwareStructuredLogger[F]]
-object LoggerFactory extends LoggerFactoryCompanion {
-  def apply[F[_]: LoggerFactory]: LoggerFactory[F] = implicitly
+trait LoggerFactory[F[_]] extends LoggerFactoryGen[F] {
```

Comment:
we can consider dropping this, since LoggerFactoryGen now has everything that is needed.

Reply:
It's kinda convinient for users who would like to use just `LoggerFactory` without the need to define LoggerType explicitly